### PR TITLE
exit(1) instead of panic

### PR DESCRIPTION
**Pull request by duglin**
Closes: #196

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: contrib/cmd/k8s-broker/k8s-broker.go [hunk 1]
```diff
@@ -55,12 +55,14 @@ func main() {
 	}
 
 	if len(options.HelmBinary) == 0 {
-		panic("Need helm_binary specified")
+		fmt.Fprintln(os.Stderr, "--helm_binary must be specified")
+		os.Exit(1)
```
Comment (krancour) [inline contrib/cmd/k8s-broker/k8s-broker.go]: Can these two lines be combined into just this?

```
log.Fatalln("--helm_binary must be specified")
```

File: contrib/cmd/k8s-broker/k8s-broker.go [hunk 2]
```diff
@@ -55,12 +55,14 @@ func main() {
...
 	}
 	r := controller.NewHelmReifier(options.HelmBinary, options.TillerHost)
 	c, err := controller.CreateController(options.RegistryHost, options.RegistryPort, r)
 	if err != nil {
-		panic(fmt.Sprintf("Error creating controller [%s]...", err.Error()))
+		fmt.Fprintf(os.Stderr, "Error creating controller: %s", err.Error())
+		os.Exit(1)
```
Comment (krancour) [inline contrib/cmd/k8s-broker/k8s-broker.go]: Can these two lines be combined into just this?

```
log.Fatalf("Error creating controller: %s\n", err)
```

File: contrib/cmd/k8s-broker/k8s-broker.go (hunk 1 above)
Comment (duglin) [inline contrib/cmd/k8s-broker/k8s-broker.go]: perhaps, but I'd really prefer if we didn't "log" command line errors. I think the output would look something like this:
```
E0121 15:46:37.334798  127442 controller.go:47] --helm_binary must be specified
```
and I don't find the user friendly. To me there's a difference between stuff written to a log file and stuff for end-user consumption - and stuff for end users doesn't usually need all of the machine readable stuff prefixing it.

Comment (krancour) [inline contrib/cmd/k8s-broker/k8s-broker.go]: @duglin in general, I agree, but the cli-ish nature of this is incidental here. Keep in mind how we expect this binary to actually be used-- in a container in a pod in a k8s cluster...

If this error occurred in such a context, you'd want to see the error timestamped, aggregated, and indexed according to the same rules as all other log messages from the same container.

So now that I think about this further, I would argue this should actually use `glog`.

Comment (duglin) [inline contrib/cmd/k8s-broker/k8s-broker.go]:
```
$ ./kubectl --qweqweqwe
Error: unknown flag: --qweqweqwe


Basic Commands (Beginner):
  create         Create a resource by filename or stdin
  expose         Take a replication controller, service, deployment or pod and expose it as a new Kubernetes Service
  run            Run a particular image on the cluster
  set            Set specific features on objects
...
```
kube-proxy, kube-dns are the same.
(yes this pains me :-)  )

Comment (duglin) [inline contrib/cmd/k8s-broker/k8s-broker.go]: I know, for other errors they do "log" it.  Don't get me started....

Comment (krancour) [inline contrib/cmd/k8s-broker/k8s-broker.go]: Not sure I understand what you were trying to say just now.

**Comment by MHBauer** [comment]
lgtm as is.

File: contrib/cmd/k8s-broker/k8s-broker.go (hunk 2 above)
Comment (pmorie) [inline contrib/cmd/k8s-broker/k8s-broker.go]: Agree, `glog.Fatalf` is preferred

**Comment by duglin** [comment]
@krancour @pmorie edits are done

**Pull request merged**